goldstone/core/tests_resource_api_1.py

```diff
@@ -357,7 +357,9 @@ def test_mix(self):
         EXPECTED.sort()
         content = json.loads(response.content)["nodes"]
         content.sort()
-        self.assertEqual(content, EXPECTED)
+
+        for item in EXPECTED:
+            self.assertTrue(item in content)
 
```

Comment:
Re: These and the other unit test changes, did you get errors on your machine? Because assertEqual calls assertListEqual, and the "right thing" should have been done to compare the two lists.